persistence.js

```diff
@@ -247,7 +247,7 @@ LevelPersistence.prototype.subscriptionsByTopic = function (pattern, cb) {
 }
 
 LevelPersistence.prototype.cleanSubscriptions = function (client, cb) {
-  var that = this
+  const that = this
```

Comment:
What's your thoughts on using arrow functions to get rid of `that` variables?

Reply:
@ralphtheninja I prefer to keep `that`, also in all other projects we use this style :)